### ISSUE TITLE
Bigtable - Condition filter doesn't return rows unless "fail" condition is set

### ISSUE THREAD

**Issue by imathews** [GoogleCloudPlatform/gcloud-node#1458]
As I understand from the API docs, the fail attribute of a condition filter should be optional, effectively being the same as providing `[{ all: false }]`. However, if I don't provide the fail attribute, I get zero rows back; the condition filter behaves as I'd expect with fail explicitly set to `[{ all: false }]`

#### Environment details
`OSX @10.11.4`
`node @6.2.0`
`npm @3.8.9`
`gcloud-node @0.37.0`

#### Steps to reproduce
````
filter = [{
	condition: {
		test: [
			{
				"column": "SEX",
				"value": "F"
			}
		],
		pass: [{
			all: true
		}],

		// Returns zero rows unless these lines are uncommented

		// fail: [
		// 	{
		// 		all: false
		// 	}
		// ]
	}
}]

table.getRows({
		filter,
		limit: 100,
}).on('data', (row) => {
    // Do things
});
````

**Comment by callmehiphop** [comment]
@imathews Thanks for reporting! I've opened a PR to remedy this (https://github.com/GoogleCloudPlatform/gcloud-node/pull/1459). Would you mind giving it a look through to ensure it resolves your issue?

**Comment by imathews** [comment]
@callmehiphop thanks for the quick fix! PR #1459 seems to solve the issue

**Comment by stephenplusplus** [comment]
The fix was merged in via https://github.com/GoogleCloudPlatform/gcloud-node/commit/aa4bf3913c0d420fdba94cfbdfdb76f970480aa3
